src/main/java/org/takes/rs/RsWithCookie.java

```diff
@@ -125,6 +145,33 @@ private static String make(final CharSequence name,
     }
 
     /**
+     * Retrieve potential previous cookie string.
+     * @param res Current Response
+     * @return Current cookie value or empty string.
+     */
+    @SuppressWarnings("PMD.OnlyOneReturn")
+    private static String getPreviousValue(final Response res) {
+        final StringBuilder cookie = new StringBuilder();
+        final StringBuilder value = new StringBuilder();
+        try {
+            for (final String header : res.head()) {
+                if (header.contains(RsWithCookie.SET_COOKIE)) {
+                    cookie.append(header);
+                    break;
+                }
+            }
+        } catch (final IOException exception) {
+            return value.toString();
```

Comment:
@Bertram25 Why are we swallowing an Exception here and then performing a return?